include/core/boundary_conditions.h

```diff
@@ -440,11 +440,11 @@ namespace BoundaryConditions
   class HTBoundaryConditions : public BoundaryConditions<dim>
   {
   public:
-    std::vector<double> value;
-    std::vector<double> h;
-    std::vector<double> Tinf;
-    std::vector<double> emissivity;
-    double              Stefan_Boltzmann_constant;
+    std::vector<std::shared_ptr<Functions::ParsedFunction<dim>>> value;
```

Comment:
maybe the value should be renamed dirichlet_value

Reply:
I agree. Did you mean only in the code or even the name of the parameter?

Reply:
in the code I think would be sufficient.